rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
```

Comment:
If I simply run `cargo fmt`, isn't that supposed to just work?

```
~/repo/ldk-node (2025-10-switch-to-nightly-rustfmt ✔) cargo fmt       
Warning: can't set `format_code_in_doc_comments = true`, unstable features are only available in nightly channel.
Warning: can't set `comment_width = 100`, unstable features are only available in nightly channel.
```

Reply:
What happens in your case if you do:
```
?main ~/workspace/ldk-node> cargo fmt --version
rustfmt 1.8.0-stable (1159e78c47 2025-09-14)
?main ~/workspace/ldk-node> git checkout 2025-10-switch-to-nightly-rustfmt
Switched to branch '2025-10-switch-to-nightly-rustfmt'
Your branch is up to date with 'origin/2025-10-switch-to-nightly-rustfmt'.
?2025-10-switch-to-nightly-rustfmt ~/workspace/ldk-node> cargo fmt --version
rustfmt 1.8.0-nightly (f6aa851dba 2025-10-07)
````

Reply:
Ah, that was it indeed. Had been reproducing issues on older rust version with the override. All good then. Farewell to the import nits 😎